### PR TITLE
Pull-Request to merge "menu-button" to the main

### PR DESCRIPTION
**Pull request by AliRezaBmeDu**
In this branch, we have made the following changes:

- Resolved the linters for html-css-javascript.
- Add responsiveness to the menu-button.
- Add script.js file to carry out the responses.

N.B. Please check this [announcement](https://microverse-students.slack.com/archives/C47D4N35M/p1686654751539179) from the microverse team concerning the webhint error.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,20 +9,28 @@
...
+            <div class="mobile-menu">
+                <button class="close-button"><img src="./icons/icons_icon-cancel.png" /></button>
+                <ul class="menu-list">
+                    <li><a href="#works" onclick="closeMenu()">Portfolio</a></li>
+                    <li><a href="#about-me" onclick="closeMenu()">About</a></li>
+                    <li><a href="#contact-form" onclick="closeMenu()">Contact</a></li>
+                </ul>
+            </div>
+
             <nav class = "toolbar-d desktop">
```
Comment (DeliceLydia) [inline index.html:34]: - [ ] I suggest that you check the padding you added to this class that is causing the app not to fit on the mobile device
Screen Shot 2023-06-26 at 21 07 20

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @AliRezaBmeDu, 

Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Good PR description
- Followed git workflow
- On mobile the hamburger menu button is visible.


## Required Changes ♻️
_Check the comments under the review._



### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (AliRezaBmeDu) [reply index.html:34]: Hello @DeliceLydia, thank you for your feedback. 
The class element you marked is for desktop-version and it is not displayed in mobile version. I don't understand how it is conflicting my mobile version. However, I updated my padding and uploaded it again.
Could you please give me a guideline?

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @AliRezaBmeDu     👋

Congratulations! 🎉 Your project is complete and ready to be merged! 🚀

image


## Highlights
- Good job with the use of GitHub features✔️ 
- Kudos on implementing the changes as requested ✅
 
 ### Verified view:
image
 
## Optional Suggestions 💡
While these comments with the [OPTIONAL] prefix won't prevent the approval of this pull request, I strongly recommend taking them into account as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really enhance your application.
Cheers, and Happy coding!👏👏👏

If you have any questions or need clarification on anything, feel free to leave a comment in the pull request thread, and don't forget to tag me so I'll receive the notification.

As per the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561), you have a limited number of reviews per project. If you feel that the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**